### PR TITLE
ref(devservices): Updating code to support new postgres container names

### PR DESCRIPTION
**Pull request by IanWoodard**
This change enables the Makefile and other logic to work with new devservices by conditionally setting the postgres container name. This enables beta users of the new devservices to perform actions such as make create-db and make drop-db among other things.

### CODE REVIEW

File: scripts/lib.sh [hunk 1]
```diff
@@ -7,6 +7,12 @@
 # shellcheck disable=SC2034 # Unused variables
 # shellcheck disable=SC2001 # https://github.com/koalaman/shellcheck/wiki/SC2001
 
+POSTGRES_CONTAINER="sentry_postgres"
+USE_NEW_DEVSERVICES=${USE_NEW_DEVSERVICES:-"0"}
+if [ "$USE_NEW_DEVSERVICES" == "1" ]; then
+POSTGRES_CONTAINER="sentry-postgres-1"
```
Comment (hubertdeng123) [inline scripts/lib.sh:13]: nit: tab/space here after if statement

File: scripts/upgrade-postgres.sh [hunk 2]
```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+POSTGRES_CONTAINER="sentry_postgres"
+USE_NEW_DEVSERVICES=${USE_NEW_DEVSERVICES:-"0"}
+if [ "$USE_NEW_DEVSERVICES" == "1" ]; then
+POSTGRES_CONTAINER="sentry-postgres-1"
```
Comment (hubertdeng123) [inline scripts/upgrade-postgres.sh:6]: here too

File: scripts/lib.sh (hunk 1 above)
Comment (IanWoodard) [reply scripts/lib.sh]: fixed

File: scripts/upgrade-postgres.sh (hunk 2 above)
Comment (IanWoodard) [reply scripts/upgrade-postgres.sh]: fixed

**Review by hubertdeng123 — APPROVED** [review]

**Pull request merged**